html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title> HTML Forms </title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+
+    <p>Email</p>
+    <label for="email">
+      <input type="text" id="email" name="email">
+    </label>
+
+    <p>Why is coding cool?</p>
```

Comment:
This should be the `label` element, it is the text labelling the form control element.